bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/custom/CommonWidgetsDPIChangeHandlers.java

```diff
@@ -11,9 +11,11 @@
  * Contributors:
  *     Yatta Solutions - initial API and implementation
  *******************************************************************************/
-package org.eclipse.swt.internal;
+package org.eclipse.swt.custom;
```

Comment:
Probably OK but let's wait for this discussion to be solved: https://github.com/eclipse-platform/eclipse.platform.swt/pull/1284/files#r1647473087

Reply:
orite